src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -637,10 +637,6 @@ Parameter Engine::GetMetric(const std::string& name, const std::map<std::string,
     return GetMetricLegacy(name, options);
 }
 
-void Engine::AddExtension(const InferenceEngine::IExtensionPtr& extension) {
-    extensionManager->AddExtension(extension);
-}
```

Comment:
Supposed add_extension API is deprecated and removed, do we also need remove ExtensionManager class? @ilyachur